README.md
# iulrp

CHANGELOG.txt
2016-02-22 15:33:49 +0100 Janusz Dobrowolski 
Fixed missing changes in upgrade sql files.
M	sql/alter2.4.sql
M	sql/alter2.4rc1.sql

2016-02-21 16:44:45 +0100 Joe Hunt 
New CHANGELOG.txt
M	CHANGELOG.txt

2016-02-21 16:43:01 +0100 Joe Hunt 
Release 2.4.RC1
M	update.html
M	version.php

2016-02-21 12:51:03 +0100 Janusz Dobrowolski 
Upgrade procedures max execution time for this version rised to 900s due to WAMP performance problems.
M	sql/alter2.4.php
M	sql/alter2.4rc1.php

2016-02-16 11:34:44 +0100 Joe Hunt 
Error in two comment lines in demo and new sql file. phpMyAdmin bailed out.
M	sql/en_US-demo.sql
M	sql/en_US-new.sql

2016-02-15 12:20:39 +0100 Joe Hunt 
More missing semicolons.
M	sql/alter2.4.sql
M	sql/alter2.4rc1.sql

2016-02-15 12:14:18 +0100 Joe Hunt 
Missing semicolon. Forgot the other file.
M	sql/alter2.4.sql

2016-02-15 11:47:39 +0100 Joe Hunt 
Missing semicolon in UPDATE instruction.
M	sql/alter2.4rc1.sql

2016-02-14 22:06:29 +0100 Janusz Dobrowolski 
Fixed upgrade procedures for manufacturing transactions, additional checks for voiding.
M	admin/db/voiding_db.inc
M	gl/view/gl_trans_view.php
M	manufacturing/includes/db/work_order_costing_db.inc
M	manufacturing/includes/db/work_order_issues_db.inc
M	manufacturing/includes/db/work_orders_db.inc
M	sql/alter2.4.sql
M	sql/alter2.4rc1.sql

2016-02-13 12:16:06 +0100 Janusz Dobrowolski 
Fixed manufacturing module voiding procedures.
M	admin/db/voiding_db.inc
M	manufacturing/includes/db/work_order_costing_db.inc
M	manufacturing/includes/db/work_order_issues_db.inc
M	manufacturing/includes/db/work_order_produce_items_db.inc
M	manufacturing/includes/db/work_orders_db.inc
M	manufacturing/work_order_costs.php

2016-01-23 12:24:28 +0100 Janusz Dobrowolski 
Fixed fixed assets stright line depreciation.
M	fixed_assets/includes/depreciation.inc

2016-01-21 10:53:33 +0100 Janusz Dobrowolski 
Fixed bug in work order view.
M	includes/types.inc

2016-01-10 10:45:45 +0100 Janusz Dobrowolski 
Manufacturing: assembly account renamed to WIP account.
M	admin/gl_setup.php
M	gl/includes/db/gl_db_accounts.inc
M	includes/db/inventory_db.inc
M	inventory/includes/db/items_category_db.inc
M	inventory/includes/db/items_db.inc
M	inventory/manage/item_categories.php
M	inventory/manage/items.php
M	manufacturing/includes/db/work_order_costing_db.inc
M	manufacturing/includes/db/work_order_issues_db.inc
M	manufacturing/includes/db/work_order_produce_items_db.inc
M	manufacturing/includes/db/work_orders_db.inc
M	sql/alter2.4.sql
M	sql/alter2.4rc1.sql
M	sql/en_US-demo.sql
M	sql/en_US-new.sql

2016-01-09 22:03:10 +0100 Janusz Dobrowolski 
Removed sparse debit account selector from Work Order Additional Costs.
M	manufacturing/includes/db/work_order_costing_db.inc
M	manufacturing/includes/db/work_order_requirements_db.inc
M	manufacturing/includes/db/work_orders_db.inc
M	manufacturing/includes/manufacturing_ui.inc
M	manufacturing/work_order_costs.php

2016-01-09 21:19:59 +0100 Janusz Dobrowolski 
Removed redundant global wip_act preference, fixed invalid account used for service BOM items.
M	admin/gl_setup.php
M	includes/data_checks.inc
M	manufacturing/includes/db/work_order_costing_db.inc
M	manufacturing/includes/db/work_order_issues_db.inc
M	manufacturing/includes/db/work_order_produce_items_db.inc
M	manufacturing/work_order_add_finished.php
M	manufacturing/work_order_issue.php
M	sql/alter2.4rc1.php
M	sql/alter2.4rc1.sql
M	sql/en_US-demo.sql
M	sql/en_US-new.sql

2016-01-21 13:34:10 +0100 Cambell Prince 
Fixed bank account balance checking procedure.
M	gl/includes/db/gl_db_bank_trans.inc
M	gl/includes/db/gl_db_banking.inc

2016-01-21 11:29:20 +0100 Cambell Prince 
Reconcile Bank Account shouldn't show void transactions.
M	gl/includes/db/gl_db_bank_accounts.inc

2016-01-09 12:44:36 +0100 Janusz Dobrowolski 
Fixed costs display in work order view.
M	manufacturing/includes/db/work_order_costing_db.inc
M	manufacturing/includes/db/work_order_issues_db.inc
M	manufacturing/includes/manufacturing_ui.inc

2016-01-14 08:02:59 +0100 Joe Hunt 
Added column Currency in Bank Accounts. Changed to all years in fixed assets.
M	includes/dashboard.inc

2016-01-13 08:23:04 +0100 Joe Hunt 
Created new fiscal year, 2016-01-01 to 2016-12-31.
M	sql/en_US-demo.sql

2016-01-02 11:08:32 +0100 Janusz Dobrowolski 
Overhead and labour cost changed to define standard costs.
M	gl/includes/db/gl_db_trans.inc
M	includes/db/inventory_db.inc
M	includes/db/manufacturing_db.inc
M	includes/ui/ui_view.inc
M	inventory/cost_update.php
M	inventory/includes/db/items_trans_db.inc
M	inventory/includes/item_adjustments_ui.inc
M	manufacturing/includes/db/work_order_costing_db.inc
M	manufacturing/includes/db/work_order_issues_db.inc
M	manufacturing/includes/db/work_order_produce_items_db.inc
M	manufacturing/includes/db/work_order_requirements_db.inc
M	manufacturing/includes/db/work_orders_db.inc
D	manufacturing/includes/db/work_orders_quick_db.inc
M	manufacturing/includes/manufacturing_db.inc
M	manufacturing/includes/manufacturing_ui.inc
M	manufacturing/includes/work_order_issue_ui.inc
M	manufacturing/view/wo_costs_view.php
M	manufacturing/work_order_costs.php
M	manufacturing/work_order_entry.php
M	purchasing/includes/db/grn_db.inc
M	purchasing/includes/purchasing_db.inc
M	reporting/rep104.php
M	reporting/rep301.php
M	reporting/rep304.php
M	sales/includes/db/sales_credit_db.inc
M	sales/includes/db/sales_delivery_db.inc
M	sales/includes/db/sales_order_db.inc
M	sales/includes/sales_db.inc
M	sales/sales_order_entry.php
M	sql/alter2.4rc1.sql

2015-12-30 15:46:52 +0100 Janusz Dobrowolski 
Removed obsolete stock_master.last_cost field.
M	fixed_assets/includes/fixed_assets_db.inc
M	inventory/includes/db/items_trans_db.inc
M	sql/alter2.4rc1.sql
M	sql/en_US-demo.sql
M	sql/en_US-new.sql

2015-12-23 21:57:33 +0100 Janusz Dobrowolski 

config_db.php
<?php

/*Connection Information for the database
$def_coy - the default company that is pre-selected on login

'host' - the computer ip address or name where the database is. The default is 'localhost' assuming that the web server is also the sql server.

'dbuser' - the user name under which the company database should be accessed.
  NB it is not secure to use root as the dbuser with no password - a user with appropriate privileges must be set up.

'dbpassword' - the password required for the dbuser to authorise the above database user.

'dbname' - the name of the database as defined in the RDMS being used. Typically RDMS allow many databases to be maintained under the same server.
'tbpref' - prefix on table names, or '' if not used. Always use non-empty prefixes if multiply company use the same database.
*/


$def_coy = 0;

$tb_pref_counter = 82;

$db_connections = array (
  0 => 
  array (
    'name' => 'ERP',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'tbpref' => 'fl94_',
  ),
  1 => 
  array (
    'name' => 'empresa01',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '49_',
  ),
  2 => 
  array (
    'name' => 'empresa02',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '50_',
  ),
  3 => 
  array (
    'name' => 'empresa03',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '51_',
  ),
  4 => 
  array (
    'name' => 'empresa04',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '52_',
  ),
  5 => 
  array (
    'name' => 'empresa05',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '53_',
  ),
  6 => 
  array (
    'name' => 'empresa06',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '54_',
  ),
  7 => 
  array (
    'name' => 'empresa07',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '55_',
  ),
  8 => 
  array (
    'name' => 'empresa08',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '56_',
  ),
  9 => 
  array (
    'name' => 'empresa09',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '57_',
  ),
  10 => 
  array (
    'name' => 'empresa10',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '58_',
  ),
  11 => 
  array (
    'name' => 'empresa11',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '59_',
  ),
  12 => 
  array (
    'name' => 'empresa12',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '60_',
  ),
  13 => 
  array (
    'name' => 'empresa13',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '61_',
  ),
  14 => 
  array (
    'name' => 'empresa14',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '62_',
  ),
  15 => 
  array (
    'name' => 'empresa15',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '63_',
  ),
  16 => 
  array (
    'name' => 'empresa16',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '64_',
  ),
  17 => 
  array (
    'name' => 'empresa17',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '65_',
  ),
  18 => 
  array (
    'name' => 'empresa18',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '66_',
  ),
  19 => 
  array (
    'name' => 'empresa19',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '67_',
  ),
  20 => 
  array (
    'name' => 'empresa20',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '68_',
  ),
  21 => 
  array (
    'name' => 'empresa21',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '69_',
  ),
  22 => 
  array (
    'name' => 'empresa22',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '70_',
  ),
  23 => 
  array (
    'name' => 'empresa23',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '71_',
  ),
  24 => 
  array (
    'name' => 'empresa24',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '72_',
  ),
  25 => 
  array (
    'name' => 'empresa25',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '73_',
  ),
  26 => 
  array (
    'name' => 'empresa26',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '74_',
  ),
  27 => 
  array (
    'name' => 'empresa27',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '75_',
  ),
  28 => 
  array (
    'name' => 'empresa28',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '76_',
  ),
  29 => 
  array (
    'name' => 'empresa29',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '77_',
  ),
  30 => 
  array (
    'name' => 'empresa30',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '78_',
  ),
  31 => 
  array (
    'name' => 'empresa31',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '79_',
  ),
  32 => 
  array (
    'name' => 'ttt',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '80_',
  ),
  33 => 
  array (
    'name' => 'ttt',
    'host' => 'localhost',
    'dbuser' => 'iultechc_fron200',
    'dbpassword' => '(47EP.6S3p',
    'dbname' => 'iultechc_fron200',
    'collation' => 'utf8_xx',
    'tbpref' => '81_',
  ),
);
